tools/sof-dump-status.py

```diff
@@ -85,8 +85,12 @@ def loadPCI(self):
                 elif tmp_output[i].split()[0].strip() == '00:':
                     tmp_line = tmp_output[i].split()
                     break
-            pci_info['hw_id']="0x" + tmp_line[2] + tmp_line[1] + " 0x" + tmp_line[4] + tmp_line[3]
-            pci_info['hw_name'] = self._pci_ids["0x" + tmp_line[4] + tmp_line[3]]
+            pci_id = "0x" + tmp_line[4] + tmp_line[3]
+            pci_info['hw_id']="0x" + tmp_line[2] + tmp_line[1] + " " + pci_id
+            if pci_id in self._pci_ids:
+                pci_info['hw_name'] = self._pci_ids[pci_id]
+            else:
+                pci_info['hw_name'] = 'unknow'
```

Comment:
please use "unknown“, not unknow

Reply:
So you think we need to cover out of intel pci device?

Reply:
My advice is only an English grammar advice.  In my opinion, all of our DUT will never have a new audio pci device except that USB one. so I think this change is a little redundant. Maybe Xiuli has something to say. @xiulipan

Reply:
it is not the USB device, the DUT is notebook,  
so have more than one PCI device of audio is reasonable